go.mod

```diff
@@ -1,4 +1,4 @@
-module github.com/giantswarm/cert-manager-app/v2
+module github.com/giantswarm/cert-manager-app/v2/v2
```

Comment:
The uninformed might believe this could be a typo. And I am uninformed.

Just curious, not important for this PR: What is the purpose of that golang code here?

Reply:
I have absolutely no idea - it's generated by the GitHub release workflow :see_no_evil: